### PR TITLE
Updates release tracking files for 1210.14.1.2 release

### PR DESCRIPTION
**Pull request by RoyShravani**
Signed-off-by: Shravani Roy <Shravani.Roy@progress.com>

## Description

Updating the following files to aid the release of 1210.14.1.2

- CHANGELOG.md
- RELEASE_NOTES.md
- VERSION
- release_tracking.txt

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -7,12 +7,18 @@ Example Note:
...
+* [azure-chef-extension #371](https://github.com/chef-partners/azure-chef-extension/pull/371)
+Documented the use of chef_server_crt option in azure-chef-extension
+* [azure-chef-extension #373](https://github.com/chef-partners/azure-chef-extension/pull/373)
+ Add files to help with release process
+* [azure-chef-extension #376](https://github.com/chef-partners/azure-chef-extension/pull/376)
+Changes for Rakefile for new process of publishing
+* [azure-chef-extension #377](https://github.com/chef-partners/azure-chef-extension/issues/377)
+Update in makefile for automating the process
+* [azure-chef-extension #378](https://github.com/chef-partners/azure-chef-extension/pull/378)
+Upgrade rack version to 2.14 for azure-chef-extension
```
Comment (dheerajd-msys) [inline RELEASE_NOTES.md:21]: We might missed mentioning PR link here for this change.

**Review by dheerajd-msys — CHANGES_REQUESTED** [review]

File: release_tracking.txt [hunk 2]
```diff
@@ -100,4 +100,12 @@
   [20210831, ubuntu, 1210.14.1.0, master, a2fa18557a65e7a17e1c5d1057f09c42d6d1f29c]
   [20210901, windows, 1210.14.1.0, master, a2fa18557a65e7a17e1c5d1057f09c42d6d1f29c]
   [20211014, ubuntu, 1210.14.1.1, master, 09a0fd63a78e2febaebd823b0abd389dcee7e7c3]
-  [20211014, windows, 1210.14.1.1, master, 09a0fd63a78e2febaebd823b0abd389dcee7e7c3]
\ No newline at end of file
+  [20211014, windows, 1210.14.1.1, master, 09a0fd63a78e2febaebd823b0abd389dcee7e7c3]
+  [20220624, windows, 1210.14.1.2, main, a5e57f5c880ceaf6ccd757193b2b555944f3cb81]
```
[the remark this answers is not in the document]
Comment (RoyShravani) [reply release_tracking.txt:104]: @dheerajd-msys  Yes, there are two entries added, it's on the next line. Please check [here](https://github.com/chef-partners/azure-chef-extension/pull/379/commits/dbc4dde3a7ea162823e3c1d7d2fc7bc634fe0de3#diff-cf7e31f104d595eda5bb9bab67cbd108e446f0c24c7b0f92ec8695b6960959c7R105) in this commit.

Apart from the rack upgrade commit, I have also added the recent commits for the makefile & rakefile as well.

File: RELEASE_NOTES.md (hunk 1 above)
Comment (RoyShravani) [reply RELEASE_NOTES.md:21]: The PR  links are added on the previous line of the description.

File: release_tracking.txt [hunk 3]
```diff
@@ -100,4 +100,12 @@
   [20210831, ubuntu, 1210.14.1.0, master, a2fa18557a65e7a17e1c5d1057f09c42d6d1f29c]
   [20210901, windows, 1210.14.1.0, master, a2fa18557a65e7a17e1c5d1057f09c42d6d1f29c]
   [20211014, ubuntu, 1210.14.1.1, master, 09a0fd63a78e2febaebd823b0abd389dcee7e7c3]
-  [20211014, windows, 1210.14.1.1, master, 09a0fd63a78e2febaebd823b0abd389dcee7e7c3]
\ No newline at end of file
+  [20211014, windows, 1210.14.1.1, master, 09a0fd63a78e2febaebd823b0abd389dcee7e7c3]
+  [20220624, windows, 1210.14.1.2, main, a5e57f5c880ceaf6ccd757193b2b555944f3cb81]
+  [20220624, ubuntu, 1210.14.1.2, main, a5e57f5c880ceaf6ccd757193b2b555944f3cb81]
+  [20220624, windows, 1210.14.1.2, main, 81039bf857c1a9ec7f0e091094689feab7e3f908]
```
[the remark this answers is not in the document]
Comment (RoyShravani) [reply release_tracking.txt]: Removed.

**Review by dheerajd-msys — APPROVED** [review]

**Review by saghoshprogress — APPROVED** [review]

**Pull request merged**